study.md

```diff
@@ -33,7 +35,10 @@ How does Ember use the URL to load view-state? Which layers in Ember are
 responsible for which tasks?
 
 ```md
-<!-- your answer here -->
+The first portion of the URL corresponds to the model, with the second portion
+corresponding to the specific method of that model. From there, the router will
+render the handlebars template of the same name as the specific method into the
+{{outlet}} of its parent route's template.
```

Comment:
Yeah, kinda. I would describe it by saying that the Router is responsible for parsing the URL and determining which Route (i.e. view state) to load; Ember then renders all of the templates at each level of nesting, from the application's root page all the way to the template specified by the Route.